__tests__/help.ts

```diff
@@ -17,6 +22,16 @@ test('npx unimported --help', (done) => {
         --flow, -f    indicates if your code is annotated with flow types    [boolean]
         --update, -u  update the ignore-lists stored in .unimportedrc.json   [boolean]"
     `);
-    done();
-  });
+  done();
```

Comment:
`done` is unnecessary when using async/await. You can remove this (make sure you remove it from the test function signature too).